src/test/ui/nll/issue-52534.stderr

```diff
@@ -0,0 +1,29 @@
+error[E0597]: `x` does not live long enough
+  --> $DIR/issue-52534.rs:22:14
+   |
+LL |     foo(|a| &x)
+   |          -   ^ `x` would have to be valid for `'0`
```

Comment:
nitpick: I'm not entirely sure where it's coming from (probably preexisting this PR), but it'd be nice to add trailing `...` here...

Reply:
Oops. When updating these messages for another review comment I accidentally removed the `...` that I added for one of your earlier reviews. Will update.

Reply:
Fixed.